### PR TITLE
Add Nunchuk guide

### PR DESCRIPTION
**Pull request by giahuy98**

### CODE REVIEW

**Comment by gStart9** [comment]
Hi [giahuy98](https://github.com/giahuy98), thanks for the PR.  Did you make the Nunchuk guide?  Last we heard, Nunchuk didn't support Tor.

**Comment by giahuy98** [comment]
Yes, I created a PR for Nunchuk guide here https://github.com/Start9Labs/electrs-wrapper/pull/20, please take a look

**Comment by kn0wmad** [comment]
We'll get it tested and get these 2 PRs reviewed tomorrow, thanks @giahuy98 :)

**Comment by gStart9** [comment]
I made some changes and attempted to push to your repo -- can you mark it a public repo that accepts PRs?

If not, that's ok, I just wanted to rearrange some stuff (mobile first, then desktop), erase the references to SSL (as much as I'd love to keep them, in no scenario does embassyOS use the ssl version for electrs), and add a note about the special orbot VPN mode requirement for Android.  Like so:

```
# Nunchuk Integration Setup

You will need to be running Tor on your device.  Refer to Start9's Tor guide:

https://start9.com/latest/user-manual/connecting/connecting-tor/tor-os/index

## Nunchuk mobile

**Note:** On Android, you will need to set up Nunchuk in VPN mode in Orbot.  Check the Android guide on the link above.

1. Open Nunchuk and select the Profile -> Network Settings

![Nunchuk mobile settings](assets/mobile-1.png)

2. Enter your electrs Hostname and Port (found in your Embassy's electrs service page, under "Properties") into the "Mainnet server" field.

![Nunchuk mobile network settings](assets/mobile-2.png)

## Nunchuk desktop

1. Open Nunchuk and go to Settings (bottom left icon) -> Network Settings

1. Enter your electrs Hostname and Port (found in your Embassy's electrs service page, under "Properties") into the "MAINNET SERVER" field.

1.  "Enable Use TOR Proxy" and then enter "127.0.0.1" for the address and "9050" for the port.

1. Save network settings then restart Nunchuk

![Nunchuk desktop](assets/desktop.png)
```

File: site/source/user-manual/service-guides/bitcoin/bitcoin-integrations.rst [hunk 1]
```diff
@@ -146,3 +146,22 @@ Trezor Suite
  **Instructions**
   
   Follow the `guide <https://github.com/Start9Labs/electrs-wrapper/blob/master/docs/integrations/trezor/guide.md>`__.
+
```
Comment (kn0wmad) [inline site/source/user-manual/service-guides/bitcoin/bitcoin-integrations.rst:149]: One nitpick - these are in alphabetical order - please move up above Samourai and we'll get this merged

**Review by kn0wmad — CHANGES_REQUESTED** [review]

File: site/source/user-manual/service-guides/bitcoin/bitcoin-integrations.rst (hunk 1 above)
Comment (giahuy98) [reply site/source/user-manual/service-guides/bitcoin/bitcoin-integrations.rst:167]: updated

**Review by kn0wmad — APPROVED** [review]

**Comment by giahuy98** [comment]
I am unable to see the updated website at https://docs.start9.com/latest/user-manual/service-guides/bitcoin/bitcoin-integrations. Do I need to take any additional steps, @kn0wmad ?

**Comment by kn0wmad** [comment]
> I am unable to see the updated website at https://docs.start9.com/latest/user-manual/service-guides/bitcoin/bitcoin-integrations. Do I need to take any additional steps, @kn0wmad ?

No, we generally get a few updates in with each publish.  I'll see what's on the docket and do a publish in the next couple days at latest

**Pull request merged**